controllers/office.js

```diff
@@ -0,0 +1,46 @@
+const OfficeModel = require('../models/office')
+// module for get all data from the office data
+module.exports.get = function get (req, res) {
+  OfficeModel.find({}).exec(function (err, offices) {
```

Comment:
you are currently using .exec() but this is not a good practice anymore, go to the mongoose documentation and find out how promises works using .then() and . catch() and change all the palaces where you are using .exec() to promises